app/models/RsFormMappings.scala

```diff
@@ -292,7 +291,7 @@ object schemeOrganiserFields {
 
 object fieldValidationPatterns {
 
-  val companyRegPattern = "(^[a-zA-Z0-9]{1,10})$"
+  val companyRegPattern = "(?i)^([0-9]\\d{6,7}|\\d{6,7}|[A-Z]{2}\\d{6})$"
```

Comment:
I know this is the regex from CTUTR but should we be letting through 6 & 7 digit numbers? From the error message I'd guess we want something like `(?i)^(\\d{8}|[A-Z]{2}\\d{6})$`

Reply:
very good spot, I didn't even read it before plopping it in - will change that bad boy

Reply:
The scheme organiser form calls the `checkLength` method so that one's ok but the company details form is relying purely on the regex for this field

Reply:
as per the discussion, leaving this as is